### PR TITLE
chore(starters): add material-kit-react to gatsby

### PR DESCRIPTION
**Pull request by WebCu**
## Description

Add Material Kit React to Gatsby

### Documentation
https://www.creative-tim.com/product/material-kit-react

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -2056,3 +2056,13 @@
     - Uses styled-components + styled-system
     - SEO with Sitemap, Schema.org JSONLD, Tags
     - Responsive images with gatsby-image
+- url: https://amazing-jones-e61bda.netlify.app/
+  repo: https://github.com/WebCu/gatsby-material-kit-react
+  description: Adaptation of Material Kit React to Gatsby
+  tags:
+    - Material Kit React
+    - Material UI
```
Comment (gillkyle) [inline docs/starters.yml:2064]:
```suggestion
    - Styling:Material
```

We have a whitelisted set of tags that are valid for starters, you can see them here: https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml

^ this list is used so the Starter showcase doesn't get bloated with all sorts of tags we can't manage, please update these to tags included in that file under `starters:` 🙂

File: docs/starters.yml [hunk 2]
```diff
@@ -2056,3 +2056,13 @@
     - Uses styled-components + styled-system
     - SEO with Sitemap, Schema.org JSONLD, Tags
     - Responsive images with gatsby-image
+- url: https://amazing-jones-e61bda.netlify.app/
+  repo: https://github.com/WebCu/gatsby-material-kit-react
+  description: Adaptation of Material Kit React to Gatsby
+  tags:
+    - Material Kit React
```
Comment (gillkyle) [inline docs/starters.yml:2063]:
```suggestion
```

**Comment by WebCu** [comment]
@gillkyle thanks for the suggestions! I'll update my the README file with clear instructions. It'll be helpful for me too :)

**Comment by gillkyle** [comment]
Cool! Feel free to ping me when you get it updated!

**Comment by marcysutton** [comment]
Hi @WebCu, I wanted to check up on this PR. Are you able to make the changes Kyle requested? Thanks!

**Comment by WebCu** [comment]
Hi @gillkyle and @marcysutton,

I have been busy and I'll continue like that for a while :/ I'm going to close this MR and when I'll have the time to do the modifications requestes I'll open it again.

Thanks for your help and your time!

**Comment by gillkyle** [comment]
Thanks @WebCu! Totally understand 🙂

**Pull request closed without merging**